src/main/java/mekanism/client/ClientTickHandler.java

```diff
@@ -77,6 +78,7 @@ public class ClientTickHandler {
     public boolean shouldReset = false;
     public static boolean firstTick = true;
     public static boolean visionEnhancement = false;
+    public static boolean hydroEnhancement = false;
```

Comment:
Unused boolean

Reply:
Thought I'd backed out all the changes in ClientTickHandler, I'm squashing to make sure I didn't miss any this time.